### DISCUSSION TITLE
Feature request: Allow confirm/decline of specific points in batches

### DISCUSSION THREAD

**Discussion by alangrafu** [Freika/dawarich#898] [Ideas]
**Describe the bug**
The visit feature is great, but I just imported my immich library and have to confirm/decline over 2K visits

<img width="1344" alt="image" src="https://github.com/user-attachments/assets/a58fe3dc-7a58-4497-bfb9-e4f23d554342">

Each confirmation takes about 1-2 seconds, so that is a lot of time. It would be great to confirm/decline all the visits to specific places in a whole batch instead, so I don't need to click on every single points requested.

**Version**
0.16.8

**To Reproduce**
1. Go to visits
2. Check the massive list you might have to confirm/decline

**Comment by Freika** [comment]
Done in 0.25.0
